.github/workflows/upstream.yml

```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   standard:
-    name: "🐍 3.11 dev • ubuntu-latest • x64"
+    name: "🐍 3.11 latest internals • ubuntu-latest • x64"
```

Comment:
Consistency would be nice, e.g.

3.11-latest above (line 33) and here.

I often look at CI logs in bulk, having to manually map `latest internals` here to `-dev` elsewhere is confusing/distracting.